docs/installation-guide/quickstart.rst

```diff
@@ -52,7 +54,8 @@ server's :file:`/etc/environment` file:
 
 Mount MetalK8s ISO
 -------------------
-Get the ISO file and mount it on the bootstrap node at this specific path.
+On your bootstrap node, download the ISO file. Mount this ISO file at the
```

Comment:
Every command should be run as root, shouldn't we remind it?

Reply:
I'll try to add `user@host $` in front of all command examples to make things clear.